src/containers/Storage/utils/index.ts

```diff
@@ -1 +1,6 @@
+import type {TVDiskStateInfo} from '../../../types/api/storage';
+
 export * from './constants';
+
+export const isFullDonorsData = (donors: TVDiskStateInfo['Donors']): donors is TVDiskStateInfo[] =>
+    Array.isArray(donors) && donors.length > 0 && 'VDiskId' in donors[0];
```

Comment:
Is this enough to check if `VDiskId` is defined only in the first donor?

Reply:
The array is either all `TVDiskStateInfo`, or all `TVSlotId`. But I suppose it would be more safe to define the field as `Array<TVDiskStateInfo | TVSlotId>` instead of `TVDiskStateInfo[] | TVSlotId[]`, and check every item. No harm in doing this, I'll update it, thanks.